WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewModel.swift

```diff
@@ -14,6 +14,10 @@ final class ShippingLabelAddressFormViewModel: NSObject {
 
     private let stores: StoresManager
 
+    var shouldShowTopBannerView: Bool {
```

Comment:
I think the name should be `shouldHideTopBannerView`, or we need to change the condition to `!=` right?

Reply:
I'd say to maintain `shouldShowTopBannerView`, so I updated the conditions 085b2de6ab6e4d77be7581bfb9406fbe7d425802